src/System.Net.Sockets/tests/FunctionalTests/SocketOptionNameTest.cs

```diff
@@ -159,9 +159,11 @@ public void MulticastInterface_Set_InvalidIndex_Throws()
         [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer))] // ActiveIssue: dotnet/corefx #29929
         public async Task MulticastInterface_Set_IPv6_AnyInterface_Succeeds()
         {
-            if (PlatformDetection.IsRedHatFamily6)
+            if (PlatformDetection.IsRedHatFamily7 || PlatformDetection.IsOSX)
             {
-                return; // [ActiveIssue(34809)]
+                // RH7 seems to have issues with multicast in Azure. Same code and setup can pass when executed outside of Azure.
+                // OSX lacks multicast routes. Probably not worth of fixing test to add them. Send fails with "no route" message.
```

Comment:
Should we conditionalize the test to only run when not in CI?  I think there's a trait for that.

Reply:
maybe we should. I was thinking about it but I did not want to create new variable. I'll lookin into it.